install/running_install.adoc

```diff
@@ -435,31 +402,78 @@ $ ansible-playbook --user openshift \
   openshift-ansible/playbooks/openstack/openshift-cluster/provision_install.yml
 ----
 
-[[running-the-advanced-installation-individual-components]]
-=== Running Individual Component Playbooks
+[[running-the-installation-playbooks]]
+=== About the installation playbooks
+
+The installer uses modularized playbooks so that administrators can install
+specific components as needed. By breaking up the roles and playbooks, there is
+better targeting of ad hoc administration tasks. This results in an increased
+level of control during installations and results in time savings. 
 
 The main installation playbook *_{pb-prefix}playbooks/deploy_cluster.yml_* runs a
 set of individual component playbooks in a specific order, and the installer
 reports back at the end what phases you have gone through. If the installation
 fails, you are notified which phase failed along with the errors from
 the Ansible run.
 
-After you resolve the errors, you can continue installation:
+////
+[IMPORTANT]
+====
+While RHEL Atomic Host is supported for running {product-title} services as
+system container, the installation method uses Ansible, which is not
+available in RHEL Atomic Host. The RPM-based installer must therefore be run
+from
+ifdef::openshift-enterprise[]
+a RHEL 7 system.
+endif::[]
+ifdef::openshift-origin[]
+a supported version of Fedora, CentOS, or RHEL.
+endif::[]
+The host initiating the installation does not need to be intended for inclusion
+in the {product-title} cluster, but it can be. Alternatively, a
+xref:running-the-advanced-installation-system-container[containerized version of the installer]
+is available as a system container, which can be run from a RHEL
+Atomic Host system.
+====
+////
+
+[[advanced-retrying-installation]]
```

Comment:
I made a new section that contains some information that didn't fit well in the existing structure. I think having a separate section about how to retry the installer might help. Please let me know if any information needs to be added.

Reply:
They were added by the console and monitoring teams. They can be added to the list or omitted I don't have a strong opinion one way or the other. If they fail then I think the error message is enough for the admin to continue without changes to the documentation.

If we want to add them I can look into the ordering and where to put them.

Reply:
We add playbooks quite frequently, I don't know what criteria we should use to determine whether or not we should document them.

Reply:
@sdodson If they're part of the core installation process, they either need to be added to the list of playbooks at ~470 or we need to remove the table and the "run the rest of the playbooks by hand" method. I'm not sure what drove the inclusion of the "run the rest of the playbooks by hand method," so I can't say which is better.

I'm on the fence about making a table to list all the playbooks that drift in and out and whether they're GA, tech preview, or dev preview/unsupported. (I think it might be a useful addition, but we'd need to talk about a sane method for maintaining it.)